compiler/src/dmd/glue.d

```diff
@@ -97,7 +97,25 @@ void generateCodeAndWrite(Module[] modules, const(char)*[] libmodules,
     if (writeLibrary)
     {
         library = Library.factory(target.objectFormat(), target.lib_ext, eSink);
-        library.setFilename(objdir, libname);
+
+        /* Determine actual file name of library to write to by combining
+         * objdir, libname, the first object file name, and lib_ext
+         */
+        const(char)[] arg;
+        if (!libname.length)
+        {
+            // get name of the first object file
+            const(char)[] n = global.params.objfiles[0].toDString;
+            n = FileName.name(n);                // remove its path
+            arg = FileName.forceExt(n, library.lib_ext); // force library name extension
+        }
+        else
```

Comment:
Inconsistent braces on this else cause.

Reply:
Don't need braces for one liners.

Reply:
Yes, but our general rule was that bracing is consistent between branches (even if one branch contains a one liner)

Reply:
It's DStyle and also DMD style based on many previous reviews.

Reply:
I don't see it in https://dlang.org/dstyle.html, and don't recall any previous reviews about it either (other than your comments from last week)

Reply:
I think it was Iain

Reply:
Regardless of stare decisis it looks terrible

Reply:
It's not my favorite either, but if you want to make work of it, perhaps add a PR to D-style so it can be discussed there. Walter told me nitpicking can be demotivating, and I agree as I've felt the same.